### PR TITLE
zh-translation:/zh/news/releases/1.4.x/announcing-1.4/change-notes/in…

### PR DESCRIPTION
**Pull request by ghl1024**
ref: [servicemesher/istio-official-translation/issues/1587](https://github.com/servicemesher/istio-official-translation/issues/1587)

### CODE REVIEW

File: content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
-## Traffic management
+## 流量管理{#traffic-management}
 
-- **Added** support for [mirroring](/zh/docs/tasks/traffic-management/mirroring/) a percentage of traffic.
-- **Improved** the Envoy sidecar. The Envoy sidecar now exits when it crashes. This change makes it easier to see whether or not the Envoy sidecar is healthy.
-- **Improved** Pilot to skip sending redundant configuration to Envoy when no changes are required.
-- **Improved** headless services to avoid conflicts with different services on the same port.
-- **Disabled** default [circuit breakers](/zh/docs/tasks/traffic-management/circuit-breaking/).
-- **Updated** the default regex engine to `re2`. Please see the [Upgrade Notes](/zh/news/releases/1.4.x/announcing-1.4/upgrade-notes) for details.
+- **新增了** 对 [mirroring](/docs/tasks/traffic-management/mirroring/) 百分比的支持。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md]:
```suggestion
- **新增了** 对 [mirroring](/docs/tasks/traffic-management/mirroring/) 百分比的流量支持。
```

File: content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
+- **改进了**  `Envoy sidecar`。 当 `Envoy sidecar` 崩溃退出时，可以更轻松地查看 `Envoy sidecar` 的状态。
+- **改进了** `Pilot` 的功能，当无请求状态时，无需进行任何更改即可跳过向 `Envoy` 发送冗余配置。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md]:
```suggestion
- **改进了** `Pilot` 的功能，当无需修改时，即可跳过向 `Envoy` 发送冗余配置的操作。
```

File: content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
+## 流量管理{#traffic-management}
 
-- **Added** support for [mirroring](/zh/docs/tasks/traffic-management/mirroring/) a percentage of traffic.
-- **Improved** the Envoy sidecar. The Envoy sidecar now exits when it crashes. This change makes it easier to see whether or not the Envoy sidecar is healthy.
-- **Improved** Pilot to skip sending redundant configuration to Envoy when no changes are required.
-- **Improved** headless services to avoid conflicts with different services on the same port.
-- **Disabled** default [circuit breakers](/zh/docs/tasks/traffic-management/circuit-breaking/).
-- **Updated** the default regex engine to `re2`. Please see the [Upgrade Notes](/zh/news/releases/1.4.x/announcing-1.4/upgrade-notes) for details.
+- **新增了** 对 [mirroring](/docs/tasks/traffic-management/mirroring/) 百分比的支持。
+- **改进了**  `Envoy sidecar`。 当 `Envoy sidecar` 崩溃退出时，可以更轻松地查看 `Envoy sidecar` 的状态。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md]:
```suggestion
- **改进了**  `Envoy sidecar`。当 `Envoy sidecar` 崩溃退出时，可以更轻松地查看 `Envoy sidecar` 的状态。
```

File: content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
 
-- **Added** experimental in-proxy telemetry reporting to [Stackdriver](https://github.com/istio/proxy/blob/{{< source_branch_name >}}/extensions/stackdriver/README.md).
-- **Improved** support for [in-proxy](/zh/docs/ops/configuration/telemetry/in-proxy-service-telemetry/) Prometheus generation of HTTP service metrics (from experimental to alpha).
-- **Improved** telemetry collection for [blocked and passthrough external service traffic](/zh/blog/2019/monitoring-external-service-traffic/).
-- **Added** the option to configure [stat patterns](/zh/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig) for Envoy stats.
-- **Added** the `inbound` and `outbound` prefixes to the Envoy HTTP stats to specify traffic direction.
-- **Improved** reporting of telemetry for traffic that goes through an egress gateway.
+- **新增了** 在 [Stackdriver](https://github.com/istio/proxy/blob/{{< source_branch_name >}}/extensions/stackdriver/README.md) 中的实验性的代理遥测报告。
+- **改进了** 对 `HTTP` 服务指标监控的[代理](/zh/docs/ops/configuration/telemetry/in-proxy-service-telemetry/) `Prometheus` ，(从实验到 `alpha`)。
+- **改进了** 遥测收集功能，用于[阻止和传递外部流量](/zh/blog/2019/monitoring-external-service-traffic/).
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md]:
```suggestion
- **改进了** 遥测收集功能，用于[阻止和传递外部流量](/zh/blog/2019/monitoring-external-service-traffic/)。
```

Comment (ghl1024) [reply content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md]: @yuxiaobo96 已修改。

File: content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md [hunk 5]
```diff
@@ -1,54 +1,54 @@
...
-- **Added** support for [mirroring](/zh/docs/tasks/traffic-management/mirroring/) a percentage of traffic.
-- **Improved** the Envoy sidecar. The Envoy sidecar now exits when it crashes. This change makes it easier to see whether or not the Envoy sidecar is healthy.
-- **Improved** Pilot to skip sending redundant configuration to Envoy when no changes are required.
-- **Improved** headless services to avoid conflicts with different services on the same port.
-- **Disabled** default [circuit breakers](/zh/docs/tasks/traffic-management/circuit-breaking/).
-- **Updated** the default regex engine to `re2`. Please see the [Upgrade Notes](/zh/news/releases/1.4.x/announcing-1.4/upgrade-notes) for details.
+- **新增了** 对 [mirroring](/docs/tasks/traffic-management/mirroring/) 百分比的流量支持。
+- **改进了**  `Envoy sidecar`。当 `Envoy sidecar` 崩溃退出时，可以更轻松地查看 `Envoy sidecar` 的状态。
+- **改进了** `Pilot` 的功能，当无需修改时，即可跳过向 `Envoy` 发送冗余配置的操作。
+- **改进了** 请求头服务，以避免与同一端口上的不同服务发生冲突。
```
Comment (gorda) [inline content/zh/news/releases/1.4.x/announcing-1.4/change-notes/index.md]: > 请求头

headless 应该是 `无头`，我觉得这里也可以不翻译：headless 服务

**Pull request merged**